src/actions/posts.js

```diff
@@ -360,7 +360,18 @@ export function getPosts(channelId, page = 0, perPage = Posts.POST_CHUNK_SIZE) {
             return null;
         }
 
-        dispatch(batchActions([
+        const actions = [];
+        if (Object.keys(posts.posts).length === Posts.POST_CHUNK_SIZE) {
```

Comment:
Won't this set the timestamp if you scroll up and load more posts in the channel? I think we only want to set it if you don't have any posts in the channel yet unless I'm mistaken